lib/Parser/OglParser.php

```diff
@@ -23,32 +23,34 @@
 class OglParser
 {
     private $hints;
+    private $matrix;
 
-    public function __construct($hints) {
+    public function __construct($hints, $matrix) {
         $this->hints = $hints;
+        $this->matrix = $matrix;
     }
 
-    public function parse($filename) {
+    public function parse($filename, $commit = null) {
         $handle = fopen($filename, "r");
```

Comment:
@MightyCreak I don't understand what you mean about false-positives?

Calling `parse($filename, null)` is a bit clunky, don't you think? If I were new to the project, I'd expect `parse($filename)` to just work, not require a second (null) parameter just because the API says so.